src/media_tools/webvtt.c

```diff
@@ -430,7 +443,6 @@ GF_Err wvtt_dump(GF_Box *a, FILE * trace)
 	GF_WebVTTSampleEntryBox *cuebox = (GF_WebVTTSampleEntryBox *)a;
 	gf_isom_box_dump_start(a, "WebVTTSampleEntryBox", trace);
 	fprintf(trace, ">\n");
-	if (cuebox->config) boxstring_dump((GF_Box *)cuebox->config, trace);
```

Comment:
Ok. Indeed, there was a change to preserve the order of boxes from the input to the dump, and for that change, we had to add all boxes in "other_boxes" instead of individual boxes. That's why we don't need to dump the 'config' here. So everything's fine. Thank you.